tests/framework/db/oci/SchemaConstraintsTest.php

```
@@ -0,0 +1,159 @@
+            "Referenced column name for foreign key '{$foreignKey}' does not match the expected value.",
+        );
+    }
+
+    public function testFindUniqueIndexes(): void
+    {
+        $db = $this->getConnection();
+
+        try {
+            $db->createCommand()->dropTable('uniqueIndex')->execute();
+        } catch (Exception) {
+        }
```

PHPMD (2.15.0)

[error] Avoid using empty try-catch blocks in testFindUniqueIndexes. (undefined)

(EmptyCatchBlock)